test/config.js

```diff
@@ -297,4 +297,248 @@ test('Config edits trigger re-bundle if using watchify', async (t) => {
 
   t.notOk(configFileString.includes('"three": 12345678'))
   t.ok(updatedConfigFileString.includes('"three": 12345678'))
-})
\ No newline at end of file
+})
+
+test('Bundle fails with first configOverride', async (t) => {
```

Comment:
improve test titles
- call out what the mistake is in the test title, eg `'invalid "resources" key'`
- "config validation fails"